packages/@atjson/renderer-hir/src/index.ts

```diff
@@ -153,16 +158,34 @@ export default class Renderer {
     );
   }
 
+  render(
```

Comment:
The use of this above in the commonmark renderer isn't totally correct... we technically still have the slices extracted (so if there's slices referenced in this, they will simply not exist, since they were pulled out already).

I understand the motivation behind this, but it has a hidden bug. (I probably have hidden bugs in other bits of code too 😱)

You may be able to make this work by only having to compile the document passed in. In the case of `render`, I think it should only accept peritext, since at this point in renderer functions, you _only_ have a peritext document for slices.